examples/gopts.c

```diff
@@ -251,6 +239,7 @@ int main (void)
 	keyDel (parentKey);
 	ksDel (ks);
 
+	// normally, you shouldn't remove the spec
```

Comment:
Because the example should clean-up after it self. We add the spec at the beginning and remove it at the end. In a real program the spec would be mounted and the program would assume it is there (and maybe verify that).